app/src/ui/check-runs/ci-check-run-list.tsx

```diff
@@ -69,11 +70,16 @@ export class CICheckRunList extends React.PureComponent<
           : props.checkRuns[0].id.toString()
     }
 
+    const checkRunEvents = new Set(
+      props.checkRuns
+        .map(c => c.actionsWorkflow?.event)
+        .filter(c => c !== undefined && c.trim() !== '')
+    )
     return {
       checkRunExpanded,
-      hasUserToggledCheckRun: currentState
-        ? currentState.hasUserToggledCheckRun
-        : false,
+      hasUserToggledCheckRun:
+        currentState !== null ? currentState.hasUserToggledCheckRun : false,
```

Comment:
Not sure if this is better or worse, but is shorter and should be equivalent 😂 
```suggestion
      hasUserToggledCheckRun: currentState?.hasUserToggledCheckRun || false,
```

Reply:
I like it!